PWGHF/D2H/Tasks/taskD0.cxx

```diff
@@ -606,6 +639,10 @@ struct HfTaskD0 {
       auto ctCandidate = hfHelper.ctD0(candidate);
       auto cpaCandidate = candidate.cpa();
       auto cpaxyCandidate = candidate.cpaXY();
+      auto trackPos = candidate.template prong0_as<TracksSelQuality>(); // positive daughter
+      auto trackNeg = candidate.template prong1_as<TracksSelQuality>(); // negative daughter
+      int minProngItsClustersFound = trackPos.itsNCls() > trackNeg.itsNCls() ? trackNeg.itsNCls() : trackPos.itsNCls();
+      int minProngTpcNCrossedRows = trackPos.tpcNClsCrossedRows() > trackNeg.tpcNClsCrossedRows() ? trackNeg.tpcNClsCrossedRows() : trackPos.tpcNClsCrossedRows();
```

Comment:
What would you call these quantities in English? Please form the names of variables accordingly.